src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/IntegrationTestSupportController.java

```diff
@@ -57,9 +64,23 @@ public Claim updateRespondByDate(
         return getClaim(claimReferenceNumber, authorisation);
     }
 
+    @PutMapping("/claims/{claimReferenceNumber}/defendant/{defendantId}")
+    public Claim linkDefendantToClaim(
+        @PathVariable("claimReferenceNumber") String claimReferenceNumber,
+        @PathVariable("defendantId") String defendantId,
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) String authorisation
+    ) {
+        Claim claim = getClaim(claimReferenceNumber, authorisation);
+
+        //todo how we should store default password? const? properties?
```

Comment:
please resolve TODO

Reply:
resolved, we no longer authenticate users, so there's no need to have password here

Reply:
I removed TODO comment as well